svthalia/concrexit
# Introduce PaymentAmountField

Closes #2091  (hopefully)

### Summary
Introduces a PaymentAmountField to be used for all payment amounts.


### How to test
Try to create payments with the interesting amounts from the test cases.

Note that I couldn't test it properly locally, because I don't run postgres and review environments are broken. And it appears as if the exact validation and error handling really matters here. The good thing is that we use the same number of digits everywhere, so create_payable will not fail at least
